<ui version="4.0" >
 <class>udpClient</class>
 <widget class="QMainWindow" name="udpClient" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>udpClient</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>26</y>
     <width>396</width>
     <height>188</height>
    </rect>
   </property>
   <widget class="QPlainTextEdit" name="outputBox" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>150</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>280</x>
      <y>150</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text" >
     <string>Send It!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>396</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionE_xit" />
   </widget>
   <addaction name="menu_File" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>214</y>
     <width>396</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionE_xit" >
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>udpClient</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
